dtschema/schemas/pci/pci-bus-common.yaml

```diff
@@ -159,6 +159,42 @@ properties:
   vpcie3v3aux-supply:
     description: 3.3v AUX regulator phandle for the slot
 
+  gen3-eq-presets:
```

Comment:
If I understand correctly, the values here would be used to program the Lane Equalization Control registers (PCIe r6.0, sec 7.7.3.4).

- There may be up to 32 of these registers (sec 7.7.3).  This seems to define only 16 elements.
- Each register is 16 bits wide; this schema seems to define 8-bit values.

Reply:
You are correct we need to define 16bits wide for each lane instead of 8-bits for gen3 equalization for gen4, gen5 etc doesn't have receiver preset hint and is only 8 bits wide. 

I will change gen3 equalization to uint16 and for remaining I will keep them as is.
we are defining per lane values. As we have maximum 16 lanes I used uint16 array. 
And there can be only 16 registers, each register represents two lane presets.